pkg/cloud/cloud.go

```diff
@@ -252,6 +257,10 @@ func (c *cloud) CreateDisk(ctx context.Context, volumeName string, diskOptions *
 	case VolumeTypeIO1, VolumeTypeIO2:
 		createType = diskOptions.VolumeType
 		iops = capacityGiB * int64(diskOptions.IOPSPerGB)
+	case VolumeTypeGP3:
+		createType = diskOptions.VolumeType
+		iops = capacityGiB * int64(diskOptions.IOPSPerGB)
```

Comment:
From my point of view I would like no connection seen between volume size and IOPS. gp3 allows you to spec any any value for IOPS once volume exceeds 32gb. It gives end user 3000IOPS base line, disconnected from the size.

Reply:
I also agree on this comment; should I
1. give another option for gp3 only such as "iopsTotal"?
2. give the "iopsTotal" option and migrate io1 and io2 iopsPerGB to iopsTotal?
3. just simply do max(3000, capacityGB * iopsPerGB)?

Reply:
+1. This is the biggest selling point of gp3. You don't need to set iops while creating a gp3 volume(see [doc](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVolume.html)], so you can simply delete this branch and add `VolumeTypeGP3` to the first case block. Line 308 below will set the request.Iops if it's passed by the user.

Reply:
But the first case uses the odd `iopsPerGB` which someone familiar with backwards compatibility would need to judge changing.

So having the new case allows changing to plain iops and throughput.

Reply:
You're right! We need to add another field "iops" so the users can directly set the value.